test.sh

```diff
@@ -175,10 +175,10 @@ end_context #test/gofmt
 # Ensure SQLite is installed so we don't recompile it each time
 go install ./Godeps/_workspace/src/github.com/mattn/go-sqlite3
 
-if [ "${TRAVIS}" == "true" ] ; then
+if [ "${TRAVIS}" == "true" ] || [ "${SKIP_UNIT_TESTS}" != "1" ] ; then
```

Comment:
I think this logic is wrong. Now the default is for local tests to run with the -cover* flags, unless SKIP_UNITTESTS is nonzero.